index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="author" content="Dagmawi Yohannes" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>portfolio</title>
+        <meta name="title" content="mobile first" />
+        <meta name="description" content="mobile first web design" />
+        <link rel="icon" type="image/png" sizes="32x32" href="images/logo.png" />
+        <link rel="stylesheet" href="css/styles.css" />
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+        <style>
+          li {
+              display: inline;
+              padding: 5px;
+          }
+        </style>
+    </head>
+    <body>
+        <section class="main">
+          <div>
+            <!-- navbar setup -->
+            <header>
+                <nav>
+                    <ul class="menu">
+                      <li>Portfolio</li>
+                      <li>About</li>
+                      <li>Contact</li>
+                    </ul>
+                    <div style="float: left;" class="button">
+                      <a href="index.html">Welcome</a>
+                    </div>
+                    <img style="float: right;" class="navbar__hamb" src="images/hamburguer.png" alt="hamburguer icon" />
```

Comment:
- [ ] Good job trying to fix the header styling as suggested by the previous reviewer 👍🏽 . Kindly ensure that the design of the toolbar in terms of the color, font, and padding, matches the figma design. currently, the header styling doesn't match the figma design 👍🏽 

your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/219807230-0a7c9bcb-352d-4fbf-9262-88d54097c46d.png)

figma design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/219807492-bfc019c5-79ec-43e7-9d95-db716cd69834.png)

Reply:
Done